### PR TITLE
Formatting fixes

### PR DESCRIPTION
**Pull request by jaxxzer**
Currently indicator widgets look funny on laptops with 4k screens.

## Before:

qgc-old-formatting

## After:

qgc-formatting-fixes-11-10pt

### CODE REVIEW

**Comment by DonLakeFlyer** [comment]
Confused on before an after. Code change seems to make widget smaller on large screen?

File: src/FlightDisplay/FlightDisplayViewWidgets.qml [hunk 1]
```diff
@@ -39,8 +39,11 @@ Item {
     QGCPalette    { id: qgcPal }
 
     function getPreferredInstrumentWidth() {
+        ScreenTools.printStats()
         if(ScreenTools.isMobile) {
             return mainWindow.width * 0.25
+        } else if(ScreenTools.isHugeScreen) {
+            return mainWindow.width * 0.11
         }
         return ScreenTools.defaultFontPixelWidth * 30
```
Comment (jaxxzer) [inline src/FlightDisplay/FlightDisplayViewWidgets.qml]: Before it was always some # of pixels * 30.
After it is 11% of screen width if using 4k screen (which has tiny pixels and apparently the same defaultFontPixelWidth as lower res screens).

Comment (DonLakeFlyer) [reply src/FlightDisplay/FlightDisplayViewWidgets.qml]: Ah, I see.

File: src/FlightDisplay/FlightDisplayViewWidgets.qml [hunk 2]
```diff
@@ -39,8 +39,11 @@ Item {
     QGCPalette    { id: qgcPal }
 
     function getPreferredInstrumentWidth() {
+        ScreenTools.printStats()
```
Comment (DonLakeFlyer) [inline src/FlightDisplay/FlightDisplayViewWidgets.qml]: Can you pull this out?

File: src/QmlControls/ScreenTools.qml [hunk 3]
```diff
@@ -90,6 +91,10 @@ Item {
         }
     }
 
+    function printStats() {
```
Comment (DonLakeFlyer) [inline src/QmlControls/ScreenTools.qml]: Can you change this to printScreenStats?

**Comment by dogmaphobic** [comment]
Has this been tested on a Mac with Retina display? Windows handles high DPI display poorly and this may be a Windows only issue. In addition, there are settings that must be configured for Qt to handle high DPI displays properly. I've been running QGC on a 4K display for over a year and have not seen this issue.

http://doc.qt.io/qt-5/highdpi.html

**Comment by jaxxzer** [comment]
Thanks for pointing that out @dogmaphobic. I will investigate this further; this was only tested on a windows display.

File: src/FlightDisplay/FlightDisplayViewWidgets.qml (hunk 1 above)
Comment (DonLakeFlyer) [reply src/FlightDisplay/FlightDisplayViewWidgets.qml]: I was confused that this isn't a mobile case.

**Comment by DonLakeFlyer** [comment]
I think this instrument width sizing needs an end to end rework. It just seems like hack on top of hack. There should be something which is combination of font sizing and percentage of window width that should be workable for everything.

**Comment by DonLakeFlyer** [comment]
Something where font size ratio is a minimum and window width percentage is a maximum.

**Comment by DonLakeFlyer** [comment]
Merging this in and then I'll test on various devices to see what things look like.

**Comment by jaxxzer** [comment]
Looks ok on early 2015 12 inch macbook with 2304x1440 display.

**Comment by jaxxzer** [comment]
Only the iMac desktops have 4k+ widths that would fall into the case where this pr changes behavior. https://en.wikipedia.org/wiki/Retina_Display

**Pull request merged**